pkg/exporter/sumologicexporter/exporter.go

```diff
@@ -440,13 +441,32 @@ func (se *sumologicexporter) pushMetricsData(ctx context.Context, md pdata.Metri
 			record.metric.CopyTo(ilms.AppendEmpty().Metrics().AppendEmpty())
 		}
 
+		errs = removeDuplicates(errs)
 		se.handleUnauthorizedErrors(ctx, errs...)
 		return consumererror.NewMetrics(multierr.Combine(errs...), droppedMetrics)
 	}
 
 	return nil
 }
 
+// removeDuplicates removes duplicate errors from a collection of errors.
+// Duplicate errors are errors that have the same message returned by `error.Error()`.
+func removeDuplicates(errs []error) []error {
+	if len(errs) < 2 {
+		return errs
+	}
+
+	var uniqueList []error
+	uniqueSet := make(map[string]bool)
```

Comment:
```suggestion
	uniqueSet := make(map[string]struct{})
```

and then use not the value but the second value returned by the map check

Reply:
Done